src/DynamoCore/Graph/Workspaces/SerializationConverters.cs

```diff
@@ -177,15 +177,15 @@ public override bool CanWrite
     /// a NodeFactory, and a Scheduler. These must be supplied at the time of 
     /// construction and should not be serialized.
     /// </summary>
-    public class WorkspaceConverter : JsonConverter
+    public class WorkspaceReadConverter : JsonConverter
```

Comment:
Hey @ikeough I'm curious what the advantage was for splitting the converters into two?